src/peft/tuners/lora/layer.py

```diff
@@ -125,9 +135,21 @@ def update_layer(
         if use_dora:
             self.dora_init(adapter_name)
             self.use_dora[adapter_name] = True
+
         else:
             self.use_dora[adapter_name] = False
 
+        if use_wlora:
+            # initialize wlora_weights
+            if not self.wlora_weights:
+                self.wlora_weights = nn.ParameterDict()
+            self.wlora_weights[adapter_name] = nn.Parameter(torch.tensor([1.0]))
```

Comment:
When this is renamed, please ensure that the parameter name still contains the substring (`lora_`), since this makes it much easier for users to quickly identify all LoRA-related parameters.